pyright_reportGeneralTypeIssues.json

```diff
@@ -1,4 +1,3 @@
-# this becomes obsolete when reportGeneralTypeIssues can be enabled in pyproject.toml
```

Comment:
Pyright uses a stricter JSON parser - comments are not part of the JSON syntax.

(There are much newer versions of pyright but there are too many things we would need to change at the moment.)